tanner/tests/test_lfi_emulator.py

```diff
@@ -10,6 +10,12 @@ def setUp(self):
         self.handler = lfi.LfiEmulator()
         self.handler.helper.host_image = 'busybox:latest'
 
+    def test_scan(self):
```

Comment:
Can you please also add the case with negative result?

Reply:
Okay, will do.